### PR TITLE
Included: Shopify shop Id in FulfilledOrderItemsSyncQueue View (#787)

### PR DESCRIPTION
**Pull request by yaminibhagat**
1. Added the Shopify Config Id in FulfilledOrderItemsSyncQueue View

### CODE REVIEW

File: entity/OmsOrderViewEntities.xml [hunk 1]
```diff
@@ -487,6 +487,9 @@ under the License.
         <member-entity entity-alias="OI" entity-name="org.apache.ofbiz.order.order.OrderItem" join-from-alias="OH">
             <key-map field-name="orderId"/>
         </member-entity>
+        <member-entity entity-alias="SHSHOPO" entity-name="co.hotwax.shopify.ShopifyShopOrder" join-from-alias="OH">
```
Comment (janvi04) [inline entity/OmsOrderViewEntities.xml:490]: I think we can name the alias as SSO.
wdyt?

Comment (janvi04) [reply entity/OmsOrderViewEntities.xml:490]: We will have to update the alias here as well while joining the entity.

**Review by janvi04 — APPROVED** [review]

File: entity/OmsOrderViewEntities.xml [hunk 2]
```diff
@@ -487,6 +487,9 @@ under the License.
         <member-entity entity-alias="OI" entity-name="org.apache.ofbiz.order.order.OrderItem" join-from-alias="OH">
             <key-map field-name="orderId"/>
         </member-entity>
+        <member-entity entity-alias="SSO" entity-name="co.hotwax.shopify.ShopifyShopOrder" join-from-alias="OH">
```
Comment (dixitdeepak) [inline entity/OmsOrderViewEntities.xml:490]: we need to make it relation-option false

Comment (yaminibhagat) [reply entity/OmsOrderViewEntities.xml]: Done! sir resolved the review comment.

**Pull request merged**